pkg/dockerutil/dockerutils.go

```diff
@@ -1904,3 +1904,19 @@ func GetContainerNames(containers []dockerTypes.Container) []string {
 func IsErrNotFound(err error) bool {
 	return dockerClient.IsErrNotFound(err)
 }
+
+// CanRunWithoutDocker returns true if the command or flag can run without Docker.
+func CanRunWithoutDocker() bool {
+	if len(os.Args) < 2 {
+		return true
+	}
+	commands := map[string]bool{
+		"-v":        true,
+		"--version": true,
+		"-h":        true,
+		"--help":    true,
+		"help":      true,
+		"hostname":  true,
+	}
+	return commands[os.Args[1]]
```

Comment:
This will work correctly (if map element not found, it returns the empty value, which is false here) but I think this might be clearer with 

```
val, ok := commands[os.Args[1]]
return ok && val 
```

Maybe not, your call. I had to go look at what happens on non-existent map entry.